src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -119,16 +123,26 @@ public override TransformWrapper Fit(IDataView input)
                 Weighting = _weighting
             };
 
-            return new TransformWrapper(Host, WordBagBuildingTransformer.Create(Host, options, input), true);
+            return new TransformWrapper(_host, WordBagBuildingTransformer.Create(_host, options, input), true);
+        }
+
+        public SchemaShape GetOutputSchema(SchemaShape inputSchema)
+        {
```

Comment:
Can you add a summary for this new public method?

Reply:
I suppose I could. While acknowledging the worth of documentation, I would point out documentation can be added any time. We have three weeks and only three weeks to finalize the public shape of the API. Anything that is there must remain there forever.

This method is not really "new" as such. It is merely copied from the existing base class, where it has precisely the same documentation as you see now.

If other people are making similar requests on other PRs, I might advise them to desist.

Reply:
I think there’s an “inheritdoc” syntax you can use. 

Aspnet uses it:

https://github.com/aspnet/AspNetCore/blob/c565386a3ed135560bc2e9017aa54a950b4e35dd/src/Mvc/Mvc.Core/src/Infrastructure/NonDisposableStream.cs

It looks like some doc tools respect these. Do ours?

Here’s the C# Lang feature request to add it as an official support in the language.

https://github.com/dotnet/csharplang/issues/313